package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "jasmine": "^2.5.1",
     "jshint": "^2.9.2",
+    "redis": "~2.6",
```

Comment:
Is this a devDependency, or a regular dependency? If it's not a regular dependency, how would a user install CHP with redis support?